### PR TITLE
Feature/mutli files gcs sink

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
This branch adds new sink that works in conjunction with https://github.com/data-integrations/multi-table-plugins/
given a database connection string and a list of tables. It will consume the tables and will write a file in GCS for each table.

This reduces the need for creating a different pipeline for each table as the current existing sink plugins.

**Comment by ghost** [comment]
I have tested avro, parquet, orc and text formats with codecs: snappy, deflate, gzip and lzo

**Comment by ghost** [comment]
@rohitsinha54 @albertshau can you please take a look?

**Comment by nitinmotgi** [comment]
@yohayg - For us to merge changes that you submit you should either sign the CCLA (
https://cla.developers.google.com/about/google-corporate) or ICLA (
https://cla.developers.google.com/about/google-individual)

Thank you for your contributions.

File: src/main/java/co/cask/gcp/common/StructuredToOrcTransformer.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                return new BooleanWritable((Boolean) fieldVal);
+            case INT:
+                return new IntWritable((Integer) fieldVal);
+            case LONG:
+                return new LongWritable((Long) fieldVal);
+            case FLOAT:
+                return new FloatWritable((Float) fieldVal);
+            case DOUBLE:
+                return new DoubleWritable((Double) fieldVal);
+            case BYTES:
```
Comment (nitinmotgi) [inline src/main/java/co/cask/gcp/common/StructuredToOrcTransformer.java:109]: Could you please include Timestamp, Datetime also. I think they are important from DB perspective.

**Comment by ghost** [comment]
Hi is ther

> @yohayg - For us to merge changes that you submit you should either sign the CCLA (
> https://cla.developers.google.com/about/google-corporate) or ICLA (
> https://cla.developers.google.com/about/google-individual)
> 
> Thank you for your contributions.

Hi, I will take a look into it. Thanks

**Pull request closed without merging**